File: C++/1114.cpp
#include <iostream>

using namespace std;

int main() {
	
	int senha;
	
	while(senha != 2002) {
		cin >> senha;
		if(senha != 2002)
			cout << "Senha Invalida" << endl;
	}
	cout << "Acesso Permitido" << endl;
	
	return 0;
}

File: JavaScript/1114.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

while (true){
    var valor = lines.shift();
    if (valor != "2002"){
        console.log("Senha Invalida");
    }
    else{
        console.log("Acesso Permitido");
        break;
    }
}